ietf/static/css/ietf.scss

```diff
@@ -47,7 +47,7 @@ url("npm:bootstrap-icons/font/fonts/bootstrap-icons.woff") format("woff");
 @import "bootstrap-icons/font/bootstrap-icons";
 
 // Leave room for fixed-top navbar...
-body {
+body.navbar-offset {
```

Comment:
This and the change to base seem out of scope for this PR subject. Please make things like this in their own PR going forward.